inst/include/population_dynamics/fleet/fleet.hpp

```diff
@@ -201,15 +208,18 @@ struct Fleet : public fims_model_object::FIMSObject<Type> {
 #ifdef TMB_MODEL
     fims_distributions::Dnorm<Type> dnorm;
     for (size_t i = 0; i < this->expected_index.size(); i++) {
-      dnorm.x = fims_math::log(this->observed_index_data->at(i));
-      dnorm.mean = fims_math::log(this->expected_index[i]);
-      dnorm.sd = fims_math::exp(this->log_obs_error[i]);
-      nll -= dnorm.evaluate(true);
+      if(this->observed_index_data->at(i) != this->observed_index_data->na_value){
```

Comment:
It seems like if we add more data types besides an index and age comps, more `if()` statements would need to be added to each of the data types' negative log likelihood assignment statements to allow for NAs to work, correct?

Reply:
correct! When coding we also discussed if we need to handle a case where there are NA values in some ages but not others despite the existence of some comp samples, but for now, you can only specify a full missing year. Are you thinking we should document this or add to an issue?